pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -139,7 +139,12 @@ public String getSegmentFile() {
     }
 
     public void deleteSegmentFile() {
-      FileUtils.deleteQuietly(new File(_segmentFile));
+      // If segment build fails with an exception (e.g. if the value of a string column contained
```

Comment:
Not sure it's really worth mentioning this specific bug as an example of an exception that can happen during segment building. If we fix the issue, for example, we'll need to update this comment.

Reply:
ok, i will remove the comment